userspace/libscap/engine/kmod/kmod.h

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 
 struct kmod_engine
 {
+	char *m_name;
```

Comment:
why don't we use a charbuf directly here instead of using `strdup`, something like `40` characters should be enough, WDYT?